src/Math-Operations-Extensions/Integer.extension.st

```diff
@@ -146,29 +146,18 @@ Integer >> isPrime [
 Integer >> isProbablyPrime [
 	"See isProbablyPrimeWithK:andQ:randomIndex: for the algoritm description."
```

Comment:
can you change typo: algorithm

Reply:
Hah, missed that.  In my defence it that typo is in the original method.